src/special-types-and-traits.md

```diff
@@ -17,34 +17,12 @@ r[lang-types.box.deref]
 * The [dereference operator] for `Box<T>` produces a place which can be moved
   from. This means that the `*` operator and the destructor of `Box<T>` are
   built-in to the language.
-
-r[lang-types.box.receiver]
-* [Methods] can take `Box<Self>` as a receiver.
-
 r[lang-types.box.fundamental]
 * A trait may be implemented for `Box<T>` in the same crate as `T`, which the
   [orphan rules] prevent for other generic types.
 
 <!-- Editor Note: This is nowhere close to an exhaustive list -->
 
-r[lang-types.rc]
-## `Rc<T>`
-
-r[lang-types.rc.receiver]
-[Methods] can take [`Rc<Self>`] as a receiver.
-
-r[lang-types.arc]
-## `Arc<T>`
-
-r[lang-types.arc.receiver]
-[Methods] can take [`Arc<Self>`] as a receiver.
-
-r[lang-types.pin]
-## `Pin<P>`
-
-r[lang-types.pin.receiver]
-[Methods] can take [`Pin<P>`] as a receiver.
```

Comment:
Nit: Is `Pin` not still a special type? A quick search in the reference reveals that it's at least [fundamental](https://doc.rust-lang.org/1.84.0/reference/glossary.html#fundamental-type-constructors), but I'd have thought it was more?